components/event-bus/api/publish/v2/error.go

```diff
@@ -0,0 +1,77 @@
+package v2
```

Comment:
Do we need this file? I find it similar to `components/event-bus/api/publish/error.go`.

Reply:
I'm trying to consolidate the APIs files, so we can easily remove the `v1` in the future